modin/pandas/test/test_io.py

```diff
@@ -71,6 +71,12 @@
 if not os.path.exists(IO_OPS_DATA_DIR):
     os.mkdir(IO_OPS_DATA_DIR)
 
+str_initial_spaces = """col1,col2,col3,col4
```

Comment:
We use `modin/pandas/test/utils.py` to store small test data. Please move it.

Reply:
Test string removed into the function it was added for as it was done for other test strings.